src/student_success_tool/analysis/pdp/dataops.py

```diff
@@ -198,6 +161,84 @@ def standardize_course_dataset(df: pd.DataFrame) -> pd.DataFrame:
     )
 
 
+def clean_up_labeled_dataset_cols_and_vals(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Drop a bunch of columns not needed or wanted for modeling, and set to null
+    any values corresponding to time after a student's current year of enrollment.
+    """
+    return (
+        df.pipe(
+            drop_columns_safely,
+            cols=[
+                # metadata
+                "institution_id",
+                "term_id",
+                "academic_year",
+                "academic_term",
```

Comment:
Adding some hypotheses in favor of keeping some of these variables, but I don't feel strongly!

Hypothesis around seasonal patterns: students enrolled in the summer look different than other terms

Reply:
I don't feel strongly, either. Let's err on the side of inclusion -- if the model finds it useful, then so be it.